Cargo.toml

```diff
@@ -17,7 +17,7 @@ features = ["bliss-audio-aubio-rs/rustdoc", "library", "ffmpeg"]
 no-default-features = true
 
 [features]
-default = ["ffmpeg", "aubio-static"]
+default = ["ffmpeg", "aubio-static", "symphonia-all"]
```

Comment:
Maybe we could leave `symphonia-all` out of default, to avoid pulling out more crates by default?

Reply:
would it be better to have the `symphonia` feature be default instead, or to not change the defaults at all?